### PR TITLE
Feature/mainnet

### PR DESCRIPTION
**Pull request by blackwatertepes**

### CODE REVIEW

File: src/accounts.js [hunk 1]
```diff
@@ -6,24 +6,27 @@ const BASE = 31; // Base 31 allows us to leave out '.', as it's used for account
 /* Private */
 
 function newAccountTransaction(name, ownerPublicKey, activePublicKey, orePayerAccountName, options = {}) {
-  const option = {
+  const { broadcast, bytes, stakedCpu, stakedNet, transfer, tokenSymbol } = {
+    broadcast: true,
     bytes: 8192,
```
Comment (traylewin) [inline src/accounts.js]: We need to make the new account staking settings configurable. The bytesRam, stakedNet, stakedCpu, and tokenSymbol should be parameters in the calling app's .env (e.g. newAccountRamBytes, newAccountStakedCpu, newAccountStakedNet).
The default values (in an .env) should be newAccountRamBytes = 2048, newAccountStakedCpu = .1, newAccountStakedNet = .1

File: src/accounts.js [hunk 2]
```diff
@@ -6,24 +6,27 @@ const BASE = 31; // Base 31 allows us to leave out '.', as it's used for account
...
-    stakedNet: 1,
+    permission: 'active',
     stakedCpu: 1,
+    stakedNet: 1,
     transfer: false,
-    ...options,
+    tokenSymbol: 'SYS',
```
Comment (traylewin) [inline src/accounts.js]: This tokenSymbol will be different depending on the target network - 'SYS' for ORE and 'EOS' for EOS networks

File: src/accounts.js [hunk 3]
```diff
@@ -6,24 +6,26 @@ const BASE = 31; // Base 31 allows us to leave out '.', as it's used for account
...
     name: 'newaccount',
     authorization: [{
       actor: orePayerAccountName,
-      permission: 'active',
+      permission,
     }],
     data: {
       creator: orePayerAccountName,
-      name,
+      newact: name, // NOTE: This will need to be changed back, once the testnets update
```
Comment (blackwatertepes) [inline src/accounts.js]: This will have to be changed, before merging into Open-Rights-Exchange

**Review by traylewin — APPROVED** [review]
Looks good!

**Pull request merged**